actions/autosync/README.md

```diff
@@ -59,8 +59,8 @@ name: Example Workflow
 
 ### Additional information on workflow inputs
 
-- `markdown_path`: This is the location for Markdown generated by the `trestle author <model>-generate` commands
-- `ssp_index_path`: This is a text file that stores the component definition information by name in trestle with the ssp name. Example below
+- `markdown_dir`: This is the location for Markdown generated by the `trestle author <model>-generate` commands
+- `ssp_index_file`: This is a text file that stores the component definition information by name in trestle with the ssp name. Example below
```

Comment:
Should the `ssp_index_file` be referenced as a .txt file or .JSON file?

Reply:
Yes, we could change this to "JSON file" for clarity.